examples/raw.rs

```diff
@@ -87,7 +87,7 @@ async fn main() -> Result<()> {
     let end = "k2";
     let pairs = client
         .with_key_only(true)
-        .scan((start..=end).to_owned(), 10)
+        .scan((start..=end).to_owned(), 10 as u32)
```

Comment:
Why change it?

Reply:
> Why change it?

May be a previous in progress need change it,can restore to the origin version now